From f974bb1f92603973711bea36a7869dfa8358aae0 Mon Sep 17 00:00:00 2001
From: Bernd Kuhls <bernd.kuhls@t-online.de>
Date: Sun, 31 Jan 2016 09:04:46 +0100
Subject: [PATCH 1/1] Fix build with musl libc

Include missing header to prevent build error detected by buildroot:
http://autobuild.buildroot.net/results/aee/aee411047265bf205f8990a3d0d2310decb5fd19/build-end.log

Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>
---
Patch sent upstream: https://github.com/olivopaolo/mtdev2tuio/pull/2

 mtdev2tuio.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/mtdev2tuio.c b/mtdev2tuio.c
index 3a9174f..f2c98d5 100644
--- a/mtdev2tuio.c
+++ b/mtdev2tuio.c
@@ -41,6 +41,7 @@
 #include <getopt.h>
 #include <signal.h>
 #include <sys/utsname.h>
+#include <sys/file.h>
 
 #define NSEC_PER_USEC   1000L
 #define NSEC_PER_SEC    1000000000L
-- 
2.7.0.rc3

